code/datums/mutations/hulk.dm

```diff
@@ -11,6 +11,10 @@
 /datum/mutation/human/hulk/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
+	if(jobban_isbanned(owner, ROLE_HULK))
+		on_losing(owner)
+		to_chat(owner, "<span class='warning'>Your muscles suddenly shrink back to normal. You remember that Nanotrasen has put genetic inhibitors on this sequence due to your misuse of the trait!</span>")
```

Comment:
Maybe just put a big red "You are banned from hulk." message or something

Reply:
It plays into the lore that NT is always one step ahead of the crew just like the US military :). I guess I can make it more something along the lines of the "gods" cursing you, since admins=gods.